bower.json

```diff
@@ -18,6 +18,12 @@
     "lazysizes": "^2.0.4",
     "imagesloaded": "^4.1.1",
     "shufflejs": "^4.0.2",
-    "swiper": "^3.4.0"
+    "swiper": "^3.4.0",
+    "zenscroll": "^3.3.0"
+  },
+  "overrides": {
```

Comment:
that scroll plugin doesnt have a proper `bower.json` so needs this for the gulp build to work